services.yaml

```diff
@@ -12,3 +12,5 @@
 - QuickSight
 - EC2
 - S3
+- AWS SDK for .NET
```

Comment:
Please drop the SDK from this list as I think it's not a useful service callout.